src/components/selectState/selectState.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import Select from "react-select";
+import useDataMaps from "../../hooks/useDataMaps";
+import { navigate } from "gatsby";
+
+import * as styles from "./selectState.module.css";
+
+export default function SelectState({
+  label,
+  link = false,
+  popoutAbbreviation,
+  setPopoutAbbrevation,
+}) {
+  const [{ stateDataMap }] = useDataMaps();
+
+  return (
+    <div className={styles.selectState}>
+      <p>{label}</p>
+      <Select
+        className={styles.selectInput}
+        value={
+          popoutAbbreviation
+            ? {
+                value: popoutAbbreviation,
+                label: stateDataMap[popoutAbbreviation]?.name,
+              }
+            : typeof link === "string"
+            ? {
+                value: link,
+                label: link,
+              }
+            : null
```

Comment:
hmm, I realize now that this is going to change dramatically with the homepage pr, because we no longer need the state popout on the homepage and this component will only link to the selected state.